### PR TITLE
Feature/laravel10

### PR DESCRIPTION
**Pull request by kitloong**

### CODE REVIEW

File: phpunit.xml [hunk 1]
```diff
@@ -1,27 +1,19 @@
...
-    </filter>
-    <php>
-        <env name="APP_ENV" value="testing"/>
-        <env name="CACHE_DRIVER" value="array"/>
-        <env name="SESSION_DRIVER" value="array"/>
-        <env name="QUEUE_DRIVER" value="sync"/>
-    </php>
-</phpunit>
\ No newline at end of file
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" backupGlobals="false" bootstrap="vendor/autoload.php" colors="true" processIsolation="false" stopOnFailure="false" xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/10.0/phpunit.xsd" cacheDirectory=".phpunit.cache" backupStaticProperties="false">
```
Comment (ikari7789) [inline phpunit.xml]: Any particular reasoning behind changing the formatting on this?

Comment (kitloong) [reply phpunit.xml]: Autogenerated with
```
./vendor/bin/phpunit --migrate-configuration
```

Sorry for being lazy, will push the better readable format

**Comment by ikari7789** [comment]
@kevincobain2000 changes more or less seem fine. But the GHE Actions only cover L7.x + PHP 7.2 but composer dependencies go back to L5.0 and PHP 7.0. Maybe fine at this point to start dropping older Laravel and PHP versions?

**Comment by kevincobain2000** [comment]
Thanks @kitloong 
I ll merge this and it won't hurt to get rid of old versions in new pull reqs.

**Pull request merged**